Add idle sample (#961)
GoogleChrome/chrome-extensions-samples: api-samples/idle/README.md

# chrome.idle

A sample that demonstrates how to use the [`chrome.idle`](https://developer.chrome.com/docs/extensions/reference/idle/) API.

## Overview

> In this sample, the `chrome.idle` API is used to detect when the user is idle and to store the history of the user's idle state.

Reviewer: ```suggestion
In this sample, the `chrome.idle` API detects and stores the history of the user's idle state.
```

## Implementation Notes

The detection interval of [`chrome.idle.onStateChanged`](https://developer.chrome.com/docs/extensions/reference/idle/#event-onStateChanged) event needs to be modified using the [`chrome.idle.setDetectionInterval`](https://developer.chrome.com/docs/extensions/reference/idle/#method-setDetectionInterval) method.

> The idle state history will be stored in the [`chrome.storage.session`](https://developer.chrome.com/docs/extensions/reference/storage/#property-session) storage area.

Reviewer: ```suggestion
The idle state history is stored in the [`chrome.storage.session`](https://developer.chrome.com/docs/extensions/reference/storage/#property-session) storage area.
```
